Documentation/install/aws/aws-terraform.md

```diff
@@ -111,6 +110,12 @@ $ cp examples/terraform.tfvars.aws build/${CLUSTER}/terraform.tfvars
 
 Edit the parameters with your AWS details, domain name, license, etc. [View all of the AWS specific options][aws-vars] and [the common Tectonic variables][vars].
 
+### Add custom TLS certificates
+
+By default, Tectonic provides self-signed certificates, which enables TLS and prevents user-provided TLS from being enabled for the cluster. To enable custom TLS certs, provide a Certificate Authority Certificate and Key (in PEM format) during Tectonic installation.
```

Comment:
s/provides/will generate self-signed certificates at install time. To enable custom TLS certs, 

#We don't gain anything by saying that we prevent user provided tls and in most cases this is not true. They can use user provided tls for ingress on their own apps all day.